backend/czi_hosted/auth/__init__.py

```diff
@@ -0,0 +1,5 @@
+# import the built in auth types so they can be registered
+
+import backend.czi_hosted.auth.auth_test  # noqa: F401
```

Comment:
My guess is that the lack of `import backend.czi_hosted_auth.auth_none` in this file is what is causing your czi-hosted unit test to fail currently.